.eslintignore

```diff
@@ -0,0 +1,4 @@
+node_modules
+test/temp
+output
+__transpiled
```

Comment:
As I see you probably use the `template-for-generator-templates` templates, am I right? 😄 You should copy content from `.gitignore` to this file :) It's a good pattern, because you will be able to have in the future `coverage` etc folders and eslint will lint these folders, what is not needed.

Reply:
Yeah, Lukasz asked to have the initial setup from `template-for-generator-templates`


> You should copy content from `.gitignore` to this file

Sure, thanks for pointing that out.